scripts/check-size-firmware.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+DTB=$1
+IMAGE_FILE_SIZE=$2
+PARTITION_NODES="/ahb/spi/flash/partitions /spi/flash/partitions"
```

Comment:
This should be generic, so passed as argument.

Reply:
Done.